frontend/javascripts/dashboard/spotlight_view.js

```diff
@@ -54,32 +53,41 @@ const WelcomeHeader = ({ history }) => (
               style={{
                 fontSize: 58,
                 textShadow: "rgba(0, 0, 0, 0.38) 0px 1px 6px",
-                textAlign: "left",
                 fontWeight: 300,
                 marginLeft: 56,
               }}
             >
               Welcome to webKnossos
             </p>
-            <p
+            <div
               style={{
-                fontSize: 20,
+                padding: "20px 60px",
                 textShadow: "rgba(0, 0, 0, 0.38) 0px 1px 6px",
                 color: "rgb(243, 243, 248)",
-                padding: "40px 60px",
-                textAlign: "left",
-                lineHeight: 1.5,
-                paddingTop: 10,
-                marginTop: 0,
               }}
             >
-              webKnossos is an in-browser annotation tool for 3D electron microscopic data that
-              facilitates user interaction with 3D image data. Together with ever better automated
-              neuron segmentations, webKnossos can push connectomics to efficient large-scale
-              reconstructions.
-            </p>
+              <p
+                style={{
+                  fontSize: 20,
+                  lineHeight: 1.5,
+                  marginTop: 0,
+                }}
+              >
+                webKnossos is an open-source tool for annotating and exploring large 3D datasets
+              </p>
+              <ul
+                style={{
+                  fontSize: 19,
+                  paddingLeft: "1.2em",
+                }}
+              >
+                <li>Fly through your data for fast skeletonization and efficiently paint voxels</li>
```

Comment:
Das "Fly" passt irgendwie noch nicht zum "voxel painting". Vielleicht so? 
`Fly through your data for fast skeletonization or/and gather 3D training/segmentation data with efficient voxel painting`?

Reply:
The text should have a similar length or it will look weird with line-break. We could split into 2 bullets.

Reply:
<img width="1605" alt="bildschirmfoto 2019-02-12 um 18 10 43" src="https://user-images.githubusercontent.com/335438/52654072-8f702c00-2ef1-11e9-878f-9555f53bd46a.png">

Reply:
stimmt. 2 bullets ist besser. ist dann halt kein Dreiklang mehr. Kann man aber evtl mit leben.

Reply:
@normanrz Did you want to push the version with the 4 bullets?

Reply:
Yep, I think that makes more sense that squeezing it into 3. I suppose @hotzenklotz actually meant `4 bullets` ;-)